migrations/versions/0467_add_limit_emails.py

```diff
@@ -0,0 +1,214 @@
+        "",
+        "Pour en savoir plus, consultez le rapport d’utilisation pour [votre service]((hyperlink_to_page_fr)).",
+        "",
+        "L’équipe Notification GC",
+        "",
+    ]
+)
+
+annual_limit_updated = "\n".join(
+    [
+        "(la version française suit)",
+        "",
+        "Hello ((name)),",
+        "",
+        "You can now send ((message_limit_en))((message_type)) messages per fiscal year.",
+        "",
+        "For more information, visit the usage report for your [service]((hyperlink_to_page_en)).",
+        "",
+        "The GC Notify Team",
+        "",
+        "---",
+        "",
+        "Bonjour ((name)),",
+        "",
+        "Vous pouvez maintenant envoyer ((message_limit_en)) ((message_type)) par exercice financier.",
+        "",
+        "Pour en savoir plus, consultez le rapport d’utilisation pour votre [service]((hyperlink_to_page_fr)).",
+        "",
+        "L’équipe Notification GC",
+    ]
+)
+
+near_annual_limit = "\n".join(
+    [
+        "(la version française suit)",
+        "",
+        "Hello ((name)),",
+        "",
+        "GC Notify has annual and daily sending limits. This message is about your annual limit.",
+        "",
+        "((service_name)) has sent ((count_en)) out of its limit of ((message_limit_en)) ((message_type)) per  fiscal year.",
+        "((service_name)) has ((remaining)) ((message_type)) remaining until April 1, ((fiscal_end)).",
+        "",
+        "For more information, visit the usage report for your [service]((hyperlink_to_page_en)).",
+        "",
+        "The GC Notify Team",
+        "",
+        "---",
+        "",
+        "Bonjour ((name)),",
+        "",
+        "Le service Notification GC comporte des limites annuelles et quotidiennes. Ce message concerne votre limite annuelle.",
+        "",
+        "((service_name)) a envoyé ((count_en)) ((message type)) et sa limite d’envoi est fixée à ((message_limit_en)) ((message type)) par exercice financier.",
+        "((service_name)) peut encore envoyer ((remaining)) ((message type)) d’ici le 1er avril ((fiscal_end)).",
+        "",
+        "Pour en savoir plus, consultez le rapport d’utilisation pour votre [service]((hyperlink_to_page_fr)).",
+        "",
+        "L’équipe Notification GC",
+        "",
+    ]
+)
+
+quarterly_annual_limit_email = "\n".join(
+    [
+        "(la version française suit)",
+        "",
+        "Hello ((name)),",
+        "",
+        "We send this report every quarter. You can use it to track the number of messages sent by your services since April 1, ((start_year)).",
+        "Review this information so you can plan to stay within the annual sending limits. Limits reset on April 1, ((end_year)).",
+        "",
+        "((data_for_each_service_en))",
+        "" "The GC Notify Team",
+        "",
+        "---",
+        "",
+        "" "Bonjour ((name)),",
```

Comment:
This should probably be:

```suggestion
        "Bonjour ((name)),",
```